src/Symfony/Bundle/WebProfilerBundle/EventListener/WebDebugToolbarListener.php

```diff
@@ -40,16 +39,14 @@ class WebDebugToolbarListener implements EventSubscriberInterface
 
     protected $twig;
     protected $urlGenerator;
-    protected $interceptRedirects;
     protected $mode;
     protected $excludedAjaxPaths;
     private $cspHandler;
 
-    public function __construct(Environment $twig, bool $interceptRedirects = false, int $mode = self::ENABLED, UrlGeneratorInterface $urlGenerator = null, string $excludedAjaxPaths = '^/bundles|^/_wdt', ContentSecurityPolicyHandler $cspHandler = null)
+    public function __construct(Environment $twig, int $mode = self::ENABLED, UrlGeneratorInterface $urlGenerator = null, string $excludedAjaxPaths = '^/bundles|^/_wdt', ContentSecurityPolicyHandler $cspHandler = null)
```

Comment:
should we have a BC layer for this signature change ? This class is not marked as `@internal`

Reply:
We should mark it as `@internal` in 4.4 IMHO.

Reply:
This class is marked as final, it should be enough ?

Reply:
@maxhelias no. BC breaks in argument position is related to using the class, not to extending it.